tests/test_data/cases_todo_in_progress.json

```diff
@@ -0,0 +1,48 @@
+[
```

Comment:
did you mean to add this?

Reply:
As in the bracket or the file?

Reply:
The file, doesn't look like it's to do with this refactor and doesn't seem like it's being used either.

Reply:
Yeah weird I feel like I remember using this at some point but apparently not, I'll remove it